### PR TITLE
[Samples] Add and update samples for tooltip feature

### LINKED ISSUE #6450: Add Tooltip Samples

### PR DESCRIPTION
**Pull request by RebeccaAnne**
# Related Issue
Fixes #6450

# Description

Added sample cards for Action.Execute.Tooltip, Action.OpenUrl.Tooltip, Action.ShowCard.Tooltip, and Action.Submit.Tooltip.

Also update existing SelectAction samples to use tooltip instead of title to populate the tooltip.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6453)

### CODE REVIEW

File: samples/v1.5/Elements/Action.Submit.Tooltip.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "type": "AdaptiveCard",
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.5",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Please enter your name below:",
+      "wrap": true
+    },
```
Comment (almedina-ms) [inline samples/v1.5/Elements/Action.Submit.Tooltip.json:10]: change for labels

File: samples/v1.0/Elements/Column.SelectAction.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
 					],
 					"selectAction": {
 						"type": "Action.OpenUrl",
-						"title": "cool link",
+						"tooltip": "cool link",
```
Comment (almedina-ms) [inline samples/v1.0/Elements/Column.SelectAction.json:28]: >"tooltip": "cool link", [](http://example.com/codeflow?start=6&length=23)

should we modify previous versions? we didn't do it with Input.* elements so we should file bugs in if we're set on following this path

Comment (RebeccaAnne) [reply samples/v1.0/Elements/Column.SelectAction.json:28]: I do believe we should be on that path. See https://github.com/microsoft/AdaptiveCards/issues/5653.

**Review by almedina-ms — APPROVED** [review]
:shipit:

File: samples/v1.2/Elements/Input.Text.InlineAction.json [hunk 3]
```diff
@@ -3,26 +3,19 @@
...
-		},
 		{
 			"type": "Input.Text",
 			"id": "iconInlineActionId",
+			"label": "Text input with an inline action",
 			"inlineAction": {
 				"type": "Action.Submit",
 				"iconUrl": "https://adaptivecards.io/content/send.png",
-				"title": "Send"
+				"tooltip": "Send"
```
Comment (jwoo-msft) [inline samples/v1.2/Elements/Input.Text.InlineAction.json:13]: Are we ok with mixing higher versions of features than the version number on the sample's folder implies?

File: samples/v1.5/Elements/Action.OpenUrl.Tooltip.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "type": "AdaptiveCard",
```
Comment (jwoo-msft) [inline samples/v1.5/Elements/Action.OpenUrl.Tooltip.json:2]: nit: tabs?

File: samples/v1.2/Elements/Input.Text.InlineAction.json (hunk 3 above)
Comment (almedina-ms) [reply samples/v1.2/Elements/Input.Text.InlineAction.json:13]: I wondered the same and it looks like that's the path we're going to follow

https://github.com/microsoft/AdaptiveCards/pull/6453#discussion_r719797898

File: samples/v1.5/Elements/Action.ShowCard.Tooltip.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        "body": [
+          {
+            "type": "TextBlock",
+            "text": "Here's some fine print for you to read. Don't break any laws! We will also not break any laws!",
+            "wrap": true,
+            "size": "Small"
+          }
+        ]
+      },
+      "tooltip": "Click here to view the details of our terms and conditions."
```
Comment (jwoo-msft) [inline samples/v1.5/Elements/Action.ShowCard.Tooltip.json:28]: nice & funny cards

File: samples/v1.5/Elements/Action.Submit.Tooltip.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+	"type": "AdaptiveCard",
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"version": "1.5",
+	"body": [
+		{
+			"label": "Please enter your name below:",
```
Comment (jwoo-msft) [inline samples/v1.5/Elements/Action.Submit.Tooltip.json:7]: nit: spacing is not uniform; tabs?

File: samples/v1.5/Elements/Action.Submit.Tooltip.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
+{
+	"type": "AdaptiveCard",
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"version": "1.5",
+	"body": [
+		{
```
Comment (jwoo-msft) [inline samples/v1.5/Elements/Action.Submit.Tooltip.json:6]: nit:spacing

**Review by jwoo-msft — APPROVED** [review]

**Pull request merged**